tests/OTAnalytics/domain/test_event.py

```diff
@@ -26,8 +26,9 @@
 )
 from OTAnalytics.domain.geometry import DirectionVector2D, ImageCoordinate
 from OTAnalytics.domain.section import SectionId
-from OTAnalytics.domain.track import Detection, PythonDetection, TrackId
+from OTAnalytics.domain.track import Detection, TrackId
 from OTAnalytics.domain.types import EventType, EventTypeParseError
+from OTAnalytics.plugin_datastore.python_track_store import PythonDetection
```

Comment:
This is a reference from the domain module into a plugin module. We could easily mock the usage in `valid_detection` to remove this dependency.